lib/foreman_bootdisk/engine.rb

```diff
@@ -57,6 +58,10 @@ class Engine < ::Rails::Engine
         provision_method 'bootdisk', N_('Boot disk based')
         template_labels 'Bootdisk' => N_('Boot disk embedded template')
         allowed_template_helpers :bootdisk_chain_url, :bootdisk_raise
+
+        extend_page "subnets/index" do |cx|
+          cx.add_pagelet :action_buttons, name: "Bootdisk", partial: "subnets/bootdisk_button"
```

Comment:
I think `action_buttons` is not a good symbol to name a pagelet. There are many "action buttons" in the app, this should have been something like `subnet_index_action_buttons`. Would you mind opening another core PR (just do refs issue number) fixing this while we can?